daprdocs/content/en/operations/hosting/kubernetes/kubernetes-production.md

```diff
@@ -68,6 +68,28 @@ Since Dapr is intended to do much of the I/O heavy lifting for your app, it's ex
 
 The CPU and memory limits above account for the fact that Dapr is intended to support a high number of I/O bound operations. It is strongly recommended that you use a monitoring tool to get a baseline for the sidecar (and app) containers and tune these settings based on those baselines.
 
+### Setting Soft Memory Limits to Dapr Sidecar
+
+It is recommended to set soft memory limits to the sidecar when you have set up memory limits. 
+This will allow the sidecar garbage collector to free up memory when the memory usage is above the limit instead of 
+waiting to be double of the last amount of memory present in the heap when it was run and that is the default behavior 
+of the [garbage collector](https://tip.golang.org/doc/gc-guide#Memory_limit) used in Go, and that can lead to OOM Kill events.
+
+For example, for an app with app-id `nodeapp`, if you have set your memory limit to be 1000Mi as mentioned above, you can use the following in your pod annotations:
```

Comment:
Mentioned above where? 
Or do you just mean "if you have set your memory limit to be 1000Mi  you can use the following in your pod annotations"

Reply:
above, I meant on the example a few lines above for production-ready setup.

```
Example settings for the Dapr sidecar in a production-ready setup:
| CPU | Memory |
|-----|--------|
| Limit: 300m, Request: 100m | Limit: 1000Mi, Request: 250Mi
```